src/benchmark/table_generator.cpp

```diff
@@ -10,7 +10,7 @@
 #include "tbb/concurrent_vector.h"
```

Comment:
Still same problem. What should I do?

Reply:
It looks like the problem was introduced with your list of revert commits. Since those removed the lines and reinserted them, they are not the same lines as before and show up as a change. I would:

- Revert to the last commit without that divergence (9d36351?)
- Cherry-Pick the commits that you intended to do
- Rebase that on top of the current master

Reply:
It sounds like a lot of work. Would it even make a difference in the end? The PR is going to be squashed anyway.